### ISSUE TITLE
feature request: find keywords

### ISSUE THREAD

**Issue by taro-ball** [bernaferrari/ChangeDetection#46]
Is it possible to add functionality to notify of change only if particular keyword is found?

**Comment by bernaferrari** [comment]
Yes, it is possible. However, I'm currently waiting for Compose to get stable to re-write this app :(

**Comment by dktzde** [comment]
Thumbs up. I could also use this feature to monitor news sites and only receive information when there is something new on the topics relevant to me.

**Comment by Quantum-Future** [comment]
> Yes, it is possible. However, I'm currently waiting for Compose to get stable to re-write this app :(

Hi, what is the current state of Compose and this project?
Do you plan to add this feature?
Thank you

**Comment by bernaferrari** [comment]
It is impossible to update the app because android changed too much. I
started rewriting in Flutter, but never finished.
